### DISCUSSION TITLE
Creating/using Use Cases

### DISCUSSION THREAD

**Discussion by twilamoon-science** [nsidc/qgreenland#746] [QGreenland-Net development]
Might we create specific use cases to drive elements of QGreenland-Net development? For example, direct partner with Asiaq re: a use case that would ensure the outcome is directly usable for them.

**Comment by MattF-NSIDC** [comment]
* Someone wants to make data more FAIR. They find some data in the wild that's not very FAIR, and they want to understand how to release a derived dataset that improves on those conditions. Making it easier to understand licensing considerations, easier to author transformation pipelines, easier to collaborate and maintain that pipeline (recipe), easier to publish the final output, easier to cite the final output (including creating references to the source data DOI if there is one)

* Someone wants to find QGIS-ready data more easily. How can we help people find data based on how they want to use it in QGIS? Some users may want data solely for visualization, and we could find them less analysis-ready online layers (e.g. WMTS). Some may want data that's analysis ready, and we could find them datasets already co-registered and at high resolution (maybe too big to include in a QGreenland zip, for example). Some may want data that's relevant to a specific area of interest, e.g. Nuuk. Perhaps they could draw a shape around that area in QGIS, triggering a query to a remote service (I forgot our 4-letter acronym :laughing:) that will find relevant data they can sift through.
